src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -576,6 +576,8 @@ properties:
       - Positive
       - Unknown
       - Not Reported
+    deprecated_enum:
+      - Indeterminant
```

Comment:
i didn't see this change specified in the ticket.

Reply:
Fixed